src/components/pager/css/Pager.css

```diff
@@ -18,41 +18,44 @@
 /* Pager UI */
 /* Class name for the list container */
 .fl-pager .fl-pager-ui {
-    margin:0; padding:0;
+    margin: 0;
+    padding: 0;
 }
 .fl-pager .fl-pager-ui li {
-    list-style-type:none;
-    display:inline;
+    list-style-type: none;
+    display: inline;
 }
 .fl-pager .fl-pager-ui a {
-    padding:0 3px;
-    margin:0 2px;
+    margin: 0 2px;
 }
 .fl-pager .fl-pager-ui .fl-pager-links a {
-    border:1px solid #ccc;
-    background-color:#fff;
-    zoom:1;
-}
-.fl-pager .fl-pager-ui .fl-pager-disabled,
-.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage {
-    color:#000;
-    border:none;
-    background-color:transparent;
-    text-decoration:none;
+    border: 1px solid #ccc;
+    background-color: #fff;
+    zoom: 1;
 }
 .fl-pager .fl-pager-ui .fl-pager-links a:hover {
-    background-color:#0287C5;
-    color:#fff;
+    background-color: #0287C5;
+    color: #fff;
     text-decoration:none;
 }
-.fl-pager .fl-pager-ui .fl-pager-disabled:hover,
+.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage,
 .fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage:hover {
-    background-color:transparent;
-    color:#000;
-    text-decoration:none;
-    cursor:default;
+    color: #fff;
+    border: none;
+    outline: 2px #000 solid;
```

Comment:
Maybe don't add the outline unless the page link has focus/hover. When it's there all the time on the current page it causes the borders of the adjacent page links to touch. Also it adds another visual indicator for which page link has focus/hover.

![screen shot 2016-04-04 at 8 57 55 am](https://cloud.githubusercontent.com/assets/556024/14248609/7c74404a-fa43-11e5-80b1-4fd8f3ee3a6a.png)